src/plugins/intel_cpu/tests/functional/shared_tests_instances/subgraph_tests/concat_resize_concat.cpp

```diff
@@ -2,27 +2,26 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
 #include <tuple>
 #include <string>
 #include <vector>
-#include "shared_test_classes/base/layer_test_utils.hpp"
-#include "ov_models/builders.hpp"
-#include <ngraph/shape.hpp>
-#include <ngraph/node.hpp>
 
-namespace SubgraphTestsDefinitions {
+namespace ov {
+namespace test {
 
-typedef std::tuple<
-        ngraph::NodeTypeInfo,                        // Node type
-        int,                                         // channels count
-        int                                          // batch count
-> ConcResizeConcParams;
+typedef std::tuple<ov::Node::type_info_t,  // Node type
+                   int,                    // channels count
+                   int                     // batch count
+                   >
+    ConcResizeConcParams;
 
 class ConcatResizeConcatTest : public testing::WithParamInterface<ConcResizeConcParams>,
-                               public LayerTestsUtils::LayerTestsCommon {
+                               public ov::test::SubgraphBaseStaticTest {
 public:
-    static std::string getTestCaseName(const testing::TestParamInfo<ConcResizeConcParams> &obj) {
-        ngraph::NodeTypeInfo resize_type;
+    static std::string getTestCaseName(const testing::TestParamInfo<ConcResizeConcParams>& obj) {
+        ov::Node::type_info_t resize_type;
```

Comment:
looks like attribute isn't used at all. Let's try to remove it from instantiation

Reply:
done